src/libs/ReportUtils.ts

```diff
@@ -3650,6 +3496,13 @@ function getReportIDFromLink(url: string | null): string {
     return reportID;
 }
 
+/**
+ * Get the report policyID given a reportID
+ */
+function getReportPolicyID(reportID: string | undefined): string | undefined {
```

Comment:
Is this the same thing? If so, I think it's more simple and standard.

```suggestion
function getReportPolicyID(reportID?: string)?: string {
```

Reply:
reverted